app/src/main/java/com/batch/mcs/finalproject/ChatUsetInteractionFragment.java

```diff
@@ -0,0 +1,108 @@
+package com.batch.mcs.finalproject;
+
+import android.content.Context;
+import android.net.Uri;
+import android.os.Bundle;
+import android.support.v4.app.Fragment;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+
+
+/**
+ * A simple {@link Fragment} subclass.
+ * Activities that contain this fragment must implement the
+ * {@link ChatUsetInteractionFragment.OnFragmentInteractionListener} interface
+ * to handle interaction events.
+ * Use the {@link ChatUsetInteractionFragment#newInstance} factory method to
+ * create an instance of this fragment.
+ */
+public class ChatUsetInteractionFragment extends Fragment {
```

Comment:
Can we clean all the code we do not need? This code should not inject to the project code that is going to be removed somewhen....  I will talk about PRs cleanliness today at training... Also, ask Joe how did he create his fragment..... he already did